cmd/brew-bootstrap-jenv-java

```diff
@@ -40,6 +40,10 @@ abort_with_shell_setup_message() {
   esac
 }
 
+abort_with_jenv_path_message() {
+  abort 'Error: the $(jenv root)/versions path does not exist. Please create it.'
```

Comment:
I think that `$(jenv root)` isn't getting evaluated here - I'm getting the error

```
$ ~/github/homebrew-bootstrap/cmd/brew-bootstrap-jenv-java
Error: the $(jenv root)/versions path does not exist. Please create it.
```